server/models/Request.model.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user: { type: mongoose.ObjectId, required: true, ref: "user" },
+    sitter: { type: mongoose.ObjectId, required: true, ref: "user" },
+    dog: { type: mongoose.ObjectId, required: true, ref: "dog" },
+    start: { type: Date, required: true, trim: true },
+    end: { type: Date, required: true, trim: true },
+    accepted: { type: Boolean },
+    declined: { type: Boolean },
```

Comment:
I find using two fields to track the payment status create unnecessary complexity and isn't very future-proof. I'd recommend reducing these to a single field (e.g. `status`) and add an `enum` validator 👍🏼